cmd/cli/config/config_test.go

```diff
@@ -3,32 +3,26 @@ package config
 import (
 	"io/ioutil"
 	"os"
-	"os/user"
 	"path"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 const (
-	configDir  = ".sonm"
-	configName = "cli.yaml"
+	testConfigDir = "/tmp/_sonm"
```

Comment:
Better to generate temporary file using https://golang.org/pkg/io/ioutil/#TempFile. This is more stable and cross-platform.